### ISSUE TITLE
Dataset folder format for PASACAL VOC dataset

### ISSUE THREAD

**Issue by mprabhakarece** [dusty-nv/pytorch-segmentation#6]
Hi,
I have downloaded VOCtrainval_11-May-2012 dataset.
Is there any pre-process for above data to run code available in this repo.
What is the folder format should be provided.
Available code is only for training or can do inference also.
I am a C++ guy. Not aware of python coding.
Kindly do help.

Thank You.

**Comment by mprabhakarece** [comment]
i am getting below error with given command below

python3 train.py data=data/VOCdevkit/VOC2012 --dataset=voc --model-dir=models --batch-size=4 --epochs=30 download=True
pytorch-segmentation/datasets/__init__.py
usage: train.py [-h] [--dataset DATASET] [-a ARCH] [--classes C] [--aux-loss]
                [--resolution N] [--width X] [--height Y] [--device DEVICE]
                [-b BATCH_SIZE] [--epochs N] [-j N] [--lr LR] [--momentum M]
                [--wd W] [--print-freq PRINT_FREQ] [--model-dir MODEL_DIR]
                [--resume RESUME] [--test-only] [--pretrained]
                [--world-size WORLD_SIZE] [--dist-url DIST_URL]
                DIR
train.py: error: unrecognized arguments: download=True
I want get clarified whether it is possible to train without pre-trained weight or it is mandatory. else  we can start fresh also?
Also kindly provide link to download pre-trained weight need to be downloaded for segmentation.

Note I didn't do any changes or pre-process to the downloaded datasets
